### DISCUSSION TITLE
MISP installation

### DISCUSSION THREAD

**Discussion by Srinivas0rch** [MISP/MISP#9578] [Q&A] [answered]
After following https://github.com/MISP/misp-docker/tree/master?tab=readme-ov-file this installed MISP. however, my doubt is do I need to create events all again from scratch? is there any MISP package which has already the events or if there is some community MISP access, how to get it?

**Comment by cvandeplas** [comment, marked as answer]
As explained here: https://www.misp-project.org/communities/

MISP is an open source software and it is also a large community of MISP users creating, maintaining and operating communities of users or organizations sharing information about threats or cyber security indicators worldwide. The MISP project doesn’t maintain an exhaustive list of all communities relying on MISP especially that some communities use MISP internally or privately.

Known Existing and Public MISP Communities:
Each community might have specific rules to join them. Below is a brief overview of existing communities, feel free to contact the respective communities that fit your organization. Some of existing public communities might be interconnected and some might be in an island mode. 

See the rest of the text [here](https://www.misp-project.org/communities/)

**Reply by Srinivas0rch** [reply]
Thank you Christophe. It helps
